src/astarte/handler.rs

```diff
@@ -59,6 +59,7 @@ pub enum DeviceError {
     Convert(#[from] MessageHubProtoError),
     /// subscriber already disconnected
     Disconnected,
+    // TODO: check if still necessary
```

Comment:
I don't think it's necessary anymore (the error), we should remove it and solve the todo